src/main/java/com/example/nlearn/controllers/GroupController.java

```diff
@@ -0,0 +1,24 @@
+package com.example.nlern.controllers;
+
+import com.example.nlern.repos.GroupRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/groups")
+public class GroupController {
+    @Autowired
+    private GroupRepository groupRepository;
```

Comment:
Controllers should not have direct access for repositories, they should only access service layer

Reply:
Done